util/cert/cert_test.go

```diff
@@ -336,6 +336,27 @@ func Test_SSHFingerprintSHA256(t *testing.T) {
 	}
 }
 
+func Test_SSHFingerPrintSHA256FromString(t *testing.T) {
+	// actual SHA256 fingerprints for keys defined above
+	fingerprints := [...]string{
+		"zzXQOXSRBEiUtuE8AikJYKwbHaxvSc0ojez9YXaGp1A",
+		"nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8",
+		"HbW3g8zUjNSksFbqTiUWPWg2Bq1x8xdGUrliXFzSnUw",
+		"eUXGGm1YGsMAS7vkcx6JOJdOGHPem5gQp4taiCfCLB8",
+		"ROQFvPThGrW4RuWLoL9tq9I9zJ42fK4XywyRtbOz/EQ",
+		"ohD8VZEXGWo6Ez8GSEJQ9WpafgLFsOfLOtGGQCQo6Og",
+		"ohD8VZEXGWo6Ez8GSEJQ9WpafgLFsOfLOtGGQCQo6Og",
+	}
+	entries, err := ParseSSHKnownHostsFromData(Test_ValidSSHKnownHostsData)
+	assert.Nil(t, err)
+	assert.Equal(t, len(entries), 7)
+	for idx, entry := range entries {
+		assert.Nil(t, err)
```

Comment:
minor - why check `err` here?

Reply:
oops, good catch. Dunno how that came there, it definitely doesn't belong there